### DISCUSSION TITLE
Welcome to Database Discussions!

### DISCUSSION THREAD

**Discussion by OpportunityLiu** [EhTagTranslation/Database#53] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by OpportunityLiu** [comment]
![image](https://user-images.githubusercontent.com/13471233/115440864-ce713900-a242-11eb-9e51-7d239e3e1c5c.png)
![image](https://user-images.githubusercontent.com/13471233/115440941-e1840900-a242-11eb-9d2f-f5e290589979.png)

**Reply by Mapaler** [reply]
这是什么啊？

**Reply by OpportunityLiu** [reply]
没啥，当图床用

**Reply by Mapaler** [reply]
但是github用的亚马逊云，经常需要挂梯子啊

**Comment by Mapaler** [comment]
好多人物介绍图片不好找，准备把这个写到wiki里去，后面就可以自己传了
https://acg18.moe/119845.html

**Reply by OpportunityLiu** [reply]
你能不能写个正常的提交消息

讨论的地方还不够多吗？需要在 commit message 里面闲聊？

**Reply by Mapaler** [reply]
要详细写改了什么吗？改了挺多的

**Reply by OpportunityLiu** [reply]
你就写“修改东方相关角色图片”就行了呗，“实验新的上传方式”算啥啊，内容、目的、动机，后面的可以省略，前面必须有啊，你可以体现动机，但是也应该先把提交的内容和目的写了

**Reply by Mapaler** [reply]
我已经把文档和新的程序都写好了，你看看怎么修订一下 [使用E绅士自身作为简介图床](../wiki/使用E绅士自身作为简介图床)

**Reply by OpportunityLiu** [reply]
就改了一下格式

**Comment by Mapaler** [comment]
为什么有很多 i.jpg.dog 的图片啊？都是原来新浪的，现在都无法显示了

**Reply by OpportunityLiu** [reply]
@EVZYH

**Reply by EVZYH** [reply]
那个图床的图片url更改了，我已经提交了pr，麻烦合并一下吧

**Reply by OpportunityLiu** [reply]
合并了

**Comment by Mapaler** [comment]
很多东方人物的标签都不存在，怎么办？

**Reply by OpportunityLiu** [reply]
不存在就去加上啊

**Reply by Mapaler** [reply]
> 
> 
> 不存在就去加上啊

不知道怎么加，我说的是E站本身

**Reply by OpportunityLiu** [reply]
![image](https://user-images.githubusercontent.com/13471233/122019270-303ec100-cdf6-11eb-9391-4dcdc09f2d3f.png)

**Reply by Mapaler** [reply]
已经换新的帖子了，帮忙看看是不是这样写？
https://forums.e-hentai.org/index.php?showtopic=246656&pid=5926761&st=337

**Comment by Mapaler** [comment]
数据库网页编辑的后退可不可以保留之前的页数，现在一点返回直接就回到了第一页。

**Reply by OpportunityLiu** [reply]
你去开个Issue吧，我有空看下

**Comment by shixichann** [comment]
Is it possible to make EhTagTranslation's database compatible with Japanese and Korean?

**Reply by OpportunityLiu** [reply]
This is not the aim of this project. Consider fork this project and adopt it to accept Japanese or Korean.

**Comment by OpportunityLiu** [comment]
@RasllKanel 提交更改前清先阅读规则 
https://github.com/EhTagTranslation/Database/wiki/%E5%8F%82%E4%B8%8E%E7%BF%BB%E8%AF%91
